atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTestSuite.java

```diff
@@ -35,14 +42,32 @@
 })
 public class CassandraTestSuite {
 
+    private final static String DOCKER_HOST_VARIABLE = "DOCKER_HOST";
+    static String CASSANDRA_HOST;
+
     @BeforeClass
-    public static void setup() throws IOException, InterruptedException {
-        CassandraService.start();
+    public static void initializeVariables() {
+        String dockerHost = System.getenv(DOCKER_HOST_VARIABLE);
```

Comment:
This is going to explode on build nodes - Linux does not set this variable. OS X will likely stop soon as well.

We have a bunch of logic for this in docker-compose-rule it would make sense to try and use.